doc/man3/EVP_PKEY_CTX_ctrl.pod

```diff
@@ -486,47 +486,47 @@ EVP_PKEY_derive() is the output of the KDF instead of the DH shared secret.
 The KDF output is typically used as a Key Encryption Key (KEK) that in turn
 encrypts a Content Encryption Key (CEK).
 
-The EVP_PKEY_CTX_set_dh_kdf_type() macro sets the key derivation function type
+The EVP_PKEY_CTX_set_dh_kdf_type() method sets the key derivation function type
```

Comment:
Fly-by note: "method" is really confusing in OpenSSL terms, since we already use that term for structure containing the set of functions for diverse implementations.

Reply:
Also, I don't quite understand why we need to qualify what those calls are.  In other words, the sentence could start like this:

```
EVP_PKEY_CTX_set_dh_kdf_type() sets ...
```

That would probably help us to not get too entangled in terminology.

Reply:
Done